common/src/main/java/com/microsoft/identity/common/internal/net/HttpRequest.java

```diff
@@ -114,9 +117,19 @@ private HttpRequest(final URL requestUrl, final Map<String, String> requestHeade
     public static HttpResponse sendPost(final URL requestUrl, final Map<String, String> requestHeaders,
                                         final byte[] requestContent, final String requestContentType)
             throws IOException {
+        Telemetry.emit(
+                new HttpStartEvent()
+                        .putMethod(REQUEST_METHOD_POST)
+                        .putPath(requestUrl)
+                        .putRequestIdHeader(requestHeaders.get("client-request-id"))
```

Comment:
Ah - same thing here about use of a constant

Reply:
Addressed